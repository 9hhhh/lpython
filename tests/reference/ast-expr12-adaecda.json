{
    "basename": "ast-expr12-adaecda",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/expr12.py",
    "infile_hash": "a28e73367996457cb261785aa6032bf884113aab4a9ec1967cd99bbc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-expr12-adaecda.stdout",
    "stdout_hash": "0969cc10090f183fb35f3629f061e3d2b379e6c522f0c831a3b80503",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}