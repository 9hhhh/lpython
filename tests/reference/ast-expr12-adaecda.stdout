(Module [(FunctionDef test ([] [(a (Name i32 Load) ()) (b (Name i32 Load) ())] [] [] [] [] []) [(Return (BinOp (Name a Load) Pow (Name b Load)))] [] (Name i32 Load) ()) (FunctionDef check ([] [] [] [] [] [] []) [(AnnAssign (Name a Store) (Name i32 Load) () 1) (Assign [(Name a Store)] (Call (Name test Load) [(ConstantInt 2 ()) (ConstantInt 2 ())] []) ()) (Return (Name a Load))] [] (Name i32 Load) ())] [])
